## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#653f2723-efae-4ef1-8527-aeea75982fb2) |
<a name="653f2723-efae-4ef1-8527-aeea75982fb2"></a>
### 11197-ad1ee92/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev8.minio.io:32407
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4e2078644ebf:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 9 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 1 minutes and 24 seconds
(4/15) Running aws-sdk-ruby tests ... done in 24 seconds
(5/15) Running awscli tests ... done in 2 minutes and 56 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 3 minutes and 55 seconds
(8/15) Running minio-dotnet tests ... done in 1 minutes and 40 seconds
(9/15) Running minio-go tests ... done in 6 minutes and 29 seconds
(10/15) Running minio-java tests ... FAILED in 9 minutes and 12 seconds
{
  "name": "minio-java",
  "function": "putObject()",
  "args": "[user metadata]",
  "duration": 178,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = AuthenticationFailed, message = -> github.com/Azure/azure-storage-blob-go/azblob.newStorageError, github.com/Azure/azure-storage-blob-go@v0.10.0/azblob/zc_storage_error.go:42\n===== RESPONSE ERROR (ServiceCode=AuthenticationFailed) =====\nDescription=Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.\nRequestId:7783affc-201e-002b-0d5d-df9956000000\nTime:2020-12-31T10:14:08.5698409Z, Details: \n   AuthenticationErrorDetail: The MAC signature found in the HTTP request 'ucUTVDa02Xvy9J1G7LLF0F38r1CMWZGPvcji72BW9uc=' is not the same as any computed signature. Server used following string to sign: 'PUT\n\n\n128\n\napplication/xml\n\n\n\n\n\n\nx-ms-blob-cache-control:\nx-ms-blob-content-disposition:\nx-ms-blob-content-encoding:\nx-ms-blob-content-language:\nx-ms-blob-content-type:application/octet-stream\nx-ms-client-request-id:452f733d-da65-4c6b-6117-168ba6bf961d\nx-ms-date:Thu, 31 Dec 2020 10:14:08 GMT\nx-ms-meta-my_header1:a   b   c\nx-ms-meta-my_header2:\"a   b   c\"\nx-ms-meta-my_project:Project One\nx-ms-meta-my_unicode_tag:å•†å“\nx-ms-version:2019-02-02\n/minioazure/minio-java-test-t24j0s/minio-java-test-2hnt3gs\ncomp:blocklist\ntimeout:1501'.\n   Code: AuthenticationFailed\n   PUT https://minioazure.blob.core.windows.net/minio-java-test-t24j0s/minio-java-test-2hnt3gs?comp=blocklist&timeout=1501\n   Authorization: REDACTED\n   Content-Length: [128]\n   Content-Type: [application/xml]\n   User-Agent: [APN/1.0 MinIO/1.0 MinIO/2020-12-31T09:41:38Z]\n   X-Ms-Blob-Cache-Control: []\n   X-Ms-Blob-Content-Disposition: []\n   X-Ms-Blob-Content-Encoding: []\n   X-Ms-Blob-Content-Language: []\n   X-Ms-Blob-Content-Type: [application/octet-stream]\n   X-Ms-Client-Request-Id: [452f733d-da65-4c6b-6117-168ba6bf961d]\n   X-Ms-Date: [Thu, 31 Dec 2020 10:14:08 GMT]\n   X-Ms-Meta-My_header1: [a   b   c]\n   X-Ms-Meta-My_header2: [\"a   b   c\"]\n   X-Ms-Meta-My_project: [Project One]\n   X-Ms-Meta-My_unicode_tag: [商品]\n   X-Ms-Version: [2019-02-02]\n   --------------------------------------------------------------------------------\n   RESPONSE Status: 403 Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.\n   Content-Length: [1091]\n   Content-Type: [application/xml]\n   Date: [Thu, 31 Dec 2020 10:14:08 GMT]\n   Server: [Microsoft-HTTPAPI/2.0]\n   X-Ms-Error-Code: [AuthenticationFailed]\n   X-Ms-Request-Id: [7783affc-201e-002b-0d5d-df9956000000]\n\n\n, bucketName = minio-java-test-t24j0s, objectName = minio-java-test-2hnt3gs, resource = /minio-java-test-t24j0s/minio-java-test-2hnt3gs, requestId = 1655C58C59BF31C1, hostId = 4d2b9e04-47fb-40ce-9ff1-f27e7bb4b1a6)\nrequest={method=PUT, url=http://minio-dev8.minio.io:32407/minio-java-test-t24j0s/minio-java-test-2hnt3gs, headers=x-amz-meta-My-Unicode-Tag: 商品\nx-amz-meta-My-Project: Project One\nx-amz-meta-My-header1: a   b   c\nx-amz-meta-My-Header2: \"a   b   c\"\nContent-Type: application/octet-stream\nHost: minio-dev8.minio.io:32407\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.0.3\nContent-MD5: A9oFTxee7YVcJ9fWsgQeKg==\nx-amz-content-sha256: 1ff7959f86334ddc5c188a5083268f600146328b2b6c5185e75bf7d9387d6b74\nx-amz-date: 20201231T101408Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20201231/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date;x-amz-meta-my-header1;x-amz-meta-my-header2;x-amz-meta-my-project;x-amz-meta-my-unicode-tag, Signature=*REDACTED*\n}\nresponse={code=403, headers=Accept-Ranges: bytes\nContent-Length: 3082\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nServer: MinIO\nVary: Origin\nX-Amz-Request-Id: 1655C58C59BF31C1\nX-Xss-Protection: 1; mode=block\nDate: Thu, 31 Dec 2020 10:14:08 GMT\n}\n >>> [io.minio.MinioClient.execute(MinioClient.java:775), io.minio.MinioClient.putObject(MinioClient.java:4547), io.minio.MinioClient.putObject(MinioClient.java:2713), io.minio.MinioClient.putObject(MinioClient.java:2830), FunctionalTest.testPutObject(FunctionalTest.java:763), FunctionalTest.putObject(FunctionalTest.java:890), FunctionalTest.runObjectTests(FunctionalTest.java:3751), FunctionalTest.runTests(FunctionalTest.java:3783), FunctionalTest.main(FunctionalTest.java:3927)]"
}
(10/15) Running minio-js tests ... done in 2 minutes and 54 seconds
(11/15) Running minio-py tests ... done in 18 minutes and 48 seconds
(12/15) Running s3cmd tests ... done in 2 minutes and 18 seconds
(13/15) Running s3select tests ... FAILED in 6 seconds
{
  "name": "s3select:test_csv_input_quote_char",
  "function": "select_object_content(bucket_name, object_name, request)",
  "args": {
    "bucket_name": "s3select-test-4054db3c-736a-4397-90b0-c7eabc66cfc7"
  },
  "duration": 6108,
  "message": "Test test_6 unexpectedly failed with: InternalError: file not found",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/s3select/csv.py\", line 42, in test_sql_api\n    for d in data.stream(10*1024):\n  File \"/usr/local/lib/python3.6/dist-packages/minio/select.py\", line 444, in stream\n    if self._read() <= 0:\n  File \"/usr/local/lib/python3.6/dist-packages/minio/select.py\", line 406, in _read\n    headers.get(\":error-code\"), headers.get(\":error-message\"),\nminio.error.MinioException: InternalError: file not found\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"./tests.py\", line 53, in main\n    test_csv_input_custom_quote_char(client, log_output)\n  File \"/mint/run/core/s3select/csv.py\", line 118, in test_csv_input_custom_quote_char\n    data, sql_opts, expected_output)\n  File \"/mint/run/core/s3select/csv.py\", line 48, in test_sql_api\n    'Test {} unexpectedly failed with: {}'.format(test_name, select_err))\nValueError: Test test_6 unexpectedly failed with: InternalError: file not found\n",
  "status": "FAIL"
}
(13/15) Running security tests ... done in 0 seconds

Executed 13 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

--- a/pkg/licverifier/verifier.go
+++ b/pkg/licverifier/verifier.go
@@ -89,7 +89,7 @@ func toLicenseInfo(claims jwt.MapClaims) (LicenseInfo, error) {
 	}
 	plan, ok := claims[plan].(string)
 	if !ok {
-		return LicenseInfo{}, errors.New("Invalid service type in claims")
+		return LicenseInfo{}, errors.New("Invalid plan in claims")
 	}
 	return LicenseInfo{
 		Email:           email,

--- a/pkg/licverifier/verifier.go
+++ b/pkg/licverifier/verifier.go
@@ -43,11 +43,11 @@ type LicenseInfo struct {
 
 // license key JSON field names
 const (
-	accountID    = "accountId"
+	accountID    = "aid"
 	sub          = "sub"
 	expiresAt    = "exp"
-	organization = "organization"
-	capacity     = "capacity"
+	organization = "org"
+	capacity     = "cap"
 	plan         = "plan"
 )
 

--- a/pkg/licverifier/verifier.go
+++ b/pkg/licverifier/verifier.go
@@ -34,21 +34,21 @@ type LicenseVerifier struct {
 // LicenseInfo holds customer metadata present in the license key.
 type LicenseInfo struct {
 	Email           string    // Email of the license key requestor
-	TeamName        string    // Subnet team name
+	Organization    string    // Subnet organization name
 	AccountID       int64     // Subnet account id
 	StorageCapacity int64     // Storage capacity used in TB
-	ServiceType     string    // Subnet service type
+	Plan            string    // Subnet plan
 	ExpiresAt       time.Time // Time of license expiry
 }
 
 // license key JSON field names
 const (
-	accountID   = "accountId"
-	sub         = "sub"
-	expiresAt   = "exp"
-	teamName    = "teamName"
-	capacity    = "capacity"
-	serviceType = "serviceType"
+	accountID    = "accountId"
+	sub          = "sub"
+	expiresAt    = "exp"
+	organization = "organization"
+	capacity     = "capacity"
+	plan         = "plan"
 )
 
 // NewLicenseVerifier returns an initialized license verifier with the given
@@ -79,24 +79,24 @@ func toLicenseInfo(claims jwt.MapClaims) (LicenseInfo, error) {
 		return LicenseInfo{}, errors.New("Invalid time of expiry in claims")
 	}
 	expiresAt := time.Unix(int64(expiryTS), 0)
-	tName, ok := claims[teamName].(string)
+	orgName, ok := claims[organization].(string)
 	if !ok {
-		return LicenseInfo{}, errors.New("Invalid team name in claims")
+		return LicenseInfo{}, errors.New("Invalid organization in claims")
 	}
 	storageCap, ok := claims[capacity].(float64)
 	if !ok {
 		return LicenseInfo{}, errors.New("Invalid storage capacity in claims")
 	}
-	sType, ok := claims[serviceType].(string)
+	plan, ok := claims[plan].(string)
 	if !ok {
 		return LicenseInfo{}, errors.New("Invalid service type in claims")
 	}
 	return LicenseInfo{
 		Email:           email,
-		TeamName:        tName,
+		Organization:    orgName,
 		AccountID:       int64(accID),
 		StorageCapacity: int64(storageCap),
-		ServiceType:     sType,
+		Plan:            plan,
 		ExpiresAt:       expiresAt,
 	}, nil
 

--- a/pkg/licverifier/verifier_test.go
+++ b/pkg/licverifier/verifier_test.go
@@ -55,20 +55,20 @@ mr/cKCUyBL7rcAvg0zNq1vcSrUSGlAmY3SEDCu3GOKnjG/U4E7+p957ocWSV+mQU
 		expectedLicInfo LicenseInfo
 		shouldPass      bool
 	}{{"", LicenseInfo{}, false},
-		{"eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJrYW5hZ2FyYWorYzFAbWluaW8uaW8iLCJleHAiOjEuNjQwOTQwMjQ1OTM3ODM5NzQ1ZTksInBsYW4iOiJTVEFOREFSRCIsImlzcyI6InN1Ym5ldEBtaW4uaW8iLCJjYXBhY2l0eSI6NTAsImlhdCI6MS42MDk0MDQyNDU5Mzc4Mzk3NDVlOSwiYWNjb3VudElkIjoxLCJvcmdhbml6YXRpb24iOiJHcmluZ290dHMgSW5jLiJ9.xflbObKf7J0GE9ljZz_r0yoir4L8MnhZ0N-oIkdivzRgL_CbnqaaLDJdn5xFY3LhRysTbn3JPnOmBrzEfVIuYDCoJm7htC3vOi0AjYr0pufsNPsKzJWzh-2pv8SeM0As", LicenseInfo{
+		{"eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJrYW5hZ2FyYWorYzFAbWluaW8uaW8iLCJjYXAiOjUwLCJvcmciOiJHcmluZ290dHMgSW5jLiIsImV4cCI6MS42NDE0NDYxNjkwMDExOTg4OTRlOSwicGxhbiI6IlNUQU5EQVJEIiwiaXNzIjoic3VibmV0QG1pbi5pbyIsImFpZCI6MSwiaWF0IjoxLjYwOTkxMDE2OTAwMTE5ODg5NGU5fQ.EhTL2xwMHnUoLQF4UR-5bjUCja3whseLU5mb9XEj7PvAae6HEIDCOMEF8Hhh20DN_v_LRE283j2ZlA5zulcXSZXS0CLcrKqbVy6QLvZfvvLuerOjJI-NBa9dSJWJ0WoN", LicenseInfo{
 			Email:           "kanagaraj+c1@minio.io",
 			Organization:    "Gringotts Inc.",
 			AccountID:       1,
 			StorageCapacity: 50,
 			Plan:            "STANDARD",
-			ExpiresAt:       time.Date(2021, time.December, 31, 8, 44, 5, 0, time.UTC),
+			ExpiresAt:       time.Date(2022, time.January, 6, 5, 16, 9, 0, time.UTC),
 		}, true},
 	}
 
 	for i, tc := range testCases {
-		// Fixing the jwt.TimeFunc at 2020-12-31 08:58:20 +0000 UTC to
+		// Fixing the jwt.TimeFunc at 2021-01-06 05:16:09 +0000 UTC to
 		// ensure that the license JWT doesn't expire ever.
-		at(time.Unix(int64(1609405100), 0), func() {
+		at(time.Unix(int64(1609910169), 0), func() {
 			licInfo, err := lv.Verify(tc.lic)
 			if err != nil && tc.shouldPass {
 				t.Fatalf("%d: Expected license to pass verification but failed with %s", i+1, err)
@@ -100,7 +100,7 @@ mr/cKCUyBL7rcAvg0zNq1vcSrUSGlAmY3SEDCu3GOKnjG/U4E7+p957ocWSV+mQU
 		fmt.Println("Failed to create license verifier", err)
 	}
 
-	licenseKey := "eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJrYW5hZ2FyYWorYzFAbWluaW8uaW8iLCJleHAiOjEuNjQwOTQwMjQ1OTM3ODM5NzQ1ZTksInBsYW4iOiJTVEFOREFSRCIsImlzcyI6InN1Ym5ldEBtaW4uaW8iLCJjYXBhY2l0eSI6NTAsImlhdCI6MS42MDk0MDQyNDU5Mzc4Mzk3NDVlOSwiYWNjb3VudElkIjoxLCJvcmdhbml6YXRpb24iOiJHcmluZ290dHMgSW5jLiJ9.xflbObKf7J0GE9ljZz_r0yoir4L8MnhZ0N-oIkdivzRgL_CbnqaaLDJdn5xFY3LhRysTbn3JPnOmBrzEfVIuYDCoJm7htC3vOi0AjYr0pufsNPsKzJWzh-2pv8SeM0As"
+	licenseKey := "eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJrYW5hZ2FyYWorYzFAbWluaW8uaW8iLCJjYXAiOjUwLCJvcmciOiJHcmluZ290dHMgSW5jLiIsImV4cCI6MS42NDE0NDYxNjkwMDExOTg4OTRlOSwicGxhbiI6IlNUQU5EQVJEIiwiaXNzIjoic3VibmV0QG1pbi5pbyIsImFpZCI6MSwiaWF0IjoxLjYwOTkxMDE2OTAwMTE5ODg5NGU5fQ.EhTL2xwMHnUoLQF4UR-5bjUCja3whseLU5mb9XEj7PvAae6HEIDCOMEF8Hhh20DN_v_LRE283j2ZlA5zulcXSZXS0CLcrKqbVy6QLvZfvvLuerOjJI-NBa9dSJWJ0WoN"
 	licInfo, err := lv.Verify(licenseKey)
 	if err != nil {
 		fmt.Println("Failed to verify license key", err)

--- a/pkg/licverifier/verifier_test.go
+++ b/pkg/licverifier/verifier_test.go
@@ -32,7 +32,7 @@ func at(t time.Time, f func()) {
 }
 
 func areEqLicenseInfo(a, b LicenseInfo) bool {
-	if a.Email == b.Email && a.TeamName == b.TeamName && a.AccountID == b.AccountID && a.ServiceType == b.ServiceType && a.StorageCapacity == b.StorageCapacity && a.ExpiresAt.Equal(b.ExpiresAt) {
+	if a.Email == b.Email && a.Organization == b.Organization && a.AccountID == b.AccountID && a.Plan == b.Plan && a.StorageCapacity == b.StorageCapacity && a.ExpiresAt.Equal(b.ExpiresAt) {
 		return true
 	}
 	return false
@@ -55,20 +55,20 @@ mr/cKCUyBL7rcAvg0zNq1vcSrUSGlAmY3SEDCu3GOKnjG/U4E7+p957ocWSV+mQU
 		expectedLicInfo LicenseInfo
 		shouldPass      bool
 	}{{"", LicenseInfo{}, false},
-		{"eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJrcCtjMUBtaW5pby5pbyIsInRlYW1OYW1lIjoiR3JpbmdvdHRzIEluYy4iLCJleHAiOjEuNjI4MjAxODYyNjgwNzM3Nzc1ZTksImNhcGFjaXR5Ijo1MCwiaWF0IjoxLjU5NjY2NTg2MjY4MDczNzc3NWU5LCJhY2NvdW50SWQiOjEsInNlcnZpY2VUeXBlIjoiU1RBTkRBUkQifQ._2EgZpjVGo3hRacO2MNavDqZoaP-hwDQ745Z-t-N6lKDwhHOzwhENb9UhiubOQ_yTJ9Ia5EqMhQrC1QCrk8-ThiftmjFGKTyYw5j7gvox_5L-R8HIegACynVlmBlF6IV", LicenseInfo{
-			Email:           "kp+c1@minio.io",
-			TeamName:        "Gringotts Inc.",
+		{"eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJrYW5hZ2FyYWorYzFAbWluaW8uaW8iLCJleHAiOjEuNjQwOTQwMjQ1OTM3ODM5NzQ1ZTksInBsYW4iOiJTVEFOREFSRCIsImlzcyI6InN1Ym5ldEBtaW4uaW8iLCJjYXBhY2l0eSI6NTAsImlhdCI6MS42MDk0MDQyNDU5Mzc4Mzk3NDVlOSwiYWNjb3VudElkIjoxLCJvcmdhbml6YXRpb24iOiJHcmluZ290dHMgSW5jLiJ9.xflbObKf7J0GE9ljZz_r0yoir4L8MnhZ0N-oIkdivzRgL_CbnqaaLDJdn5xFY3LhRysTbn3JPnOmBrzEfVIuYDCoJm7htC3vOi0AjYr0pufsNPsKzJWzh-2pv8SeM0As", LicenseInfo{
+			Email:           "kanagaraj+c1@minio.io",
+			Organization:    "Gringotts Inc.",
 			AccountID:       1,
 			StorageCapacity: 50,
-			ServiceType:     "STANDARD",
-			ExpiresAt:       time.Date(2021, time.August, 5, 15, 17, 42, 0, time.FixedZone("PDT", -7*60*60)),
+			Plan:            "STANDARD",
+			ExpiresAt:       time.Date(2021, time.December, 31, 8, 44, 5, 0, time.UTC),
 		}, true},
 	}
 
 	for i, tc := range testCases {
-		// Fixing the jwt.TimeFunc at 2020-08-05 22:17:43 +0000 UTC to
+		// Fixing the jwt.TimeFunc at 2020-12-31 08:58:20 +0000 UTC to
 		// ensure that the license JWT doesn't expire ever.
-		at(time.Unix(int64(1596665863), 0), func() {
+		at(time.Unix(int64(1609405100), 0), func() {
 			licInfo, err := lv.Verify(tc.lic)
 			if err != nil && tc.shouldPass {
 				t.Fatalf("%d: Expected license to pass verification but failed with %s", i+1, err)
@@ -100,7 +100,7 @@ mr/cKCUyBL7rcAvg0zNq1vcSrUSGlAmY3SEDCu3GOKnjG/U4E7+p957ocWSV+mQU
 		fmt.Println("Failed to create license verifier", err)
 	}
 
-	licenseKey := "eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJrcCtjMUBtaW5pby5pbyIsInRlYW1OYW1lIjoiR3JpbmdvdHRzIEluYy4iLCJleHAiOjEuNjI4MjAxODYyNjgwNzM3Nzc1ZTksImNhcGFjaXR5Ijo1MCwiaWF0IjoxLjU5NjY2NTg2MjY4MDczNzc3NWU5LCJhY2NvdW50SWQiOjEsInNlcnZpY2VUeXBlIjoiU1RBTkRBUkQifQ._2EgZpjVGo3hRacO2MNavDqZoaP-hwDQ745Z-t-N6lKDwhHOzwhENb9UhiubOQ_yTJ9Ia5EqMhQrC1QCrk8-ThiftmjFGKTyYw5j7gvox_5L-R8HIegACynVlmBlF6IV"
+	licenseKey := "eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJrYW5hZ2FyYWorYzFAbWluaW8uaW8iLCJleHAiOjEuNjQwOTQwMjQ1OTM3ODM5NzQ1ZTksInBsYW4iOiJTVEFOREFSRCIsImlzcyI6InN1Ym5ldEBtaW4uaW8iLCJjYXBhY2l0eSI6NTAsImlhdCI6MS42MDk0MDQyNDU5Mzc4Mzk3NDVlOSwiYWNjb3VudElkIjoxLCJvcmdhbml6YXRpb24iOiJHcmluZ290dHMgSW5jLiJ9.xflbObKf7J0GE9ljZz_r0yoir4L8MnhZ0N-oIkdivzRgL_CbnqaaLDJdn5xFY3LhRysTbn3JPnOmBrzEfVIuYDCoJm7htC3vOi0AjYr0pufsNPsKzJWzh-2pv8SeM0As"
 	licInfo, err := lv.Verify(licenseKey)
 	if err != nil {
 		fmt.Println("Failed to verify license key", err)

